packages/overlay/test/overlay-trigger-extended.test.ts

```diff
@@ -247,12 +255,21 @@ describe('Overlay Trigger - extended', () => {
 
         await sendMouseTo(textfield, 'click');
 
+        // Wait a bit for the click to register
+        await aTimeout(50);
+
+        // If click didn't work, try programmatic focus
```

Comment:
Before the overlay is opened, I think it’s totally fine to use programmatic focus to assert that the textfield is normally focusable. But doing the same after the overlay is open feels like it could cause the last assertion (below) to always pass, even if the overlay is still occluding the textfield.

Reply:
Good point @rubencarvalho We are unfortunately bypassing the occlusion check.